### PR TITLE
improved specificity on did resolution diagrams

### PR DESCRIPTION
**Pull request by andorsk**
According to https://www.w3.org/TR/did-core/#did-resolution, did resolution follows the following interface: 

```
resolve(did, resolutionOptions) →
   « didResolutionMetadata, didDocument, didDocumentMetadata »
```

I've added the objects didResolutionMetadata, didDocumentMetadata to the diagrams with the appropriate interfaces. 

Thank you for the review!

### CODE REVIEW

**Comment by andorsk** [comment]
@OR13 would it be possible to do a review on this? I can't assign reviewers.

**Review by OR13 — APPROVED** [review]

**Comment by andorsk** [comment]
@OR13 Thanks for approving! I don't have write access either. Can you please merge this if this is good to go?

**Comment by andorsk** [comment]
Thanks!

**Pull request merged**
